rpcd/playbooks/roles/rpc_pre_upgrade/templates/status.txt.j2

```diff
@@ -25,6 +25,11 @@ cinder service-list output as at {{ ansible_date_time.date }}-{{ ansible_date_ti
 {{ line }}
 {% endfor %}
 
+orchestration service-list output as at {{ ansible_date_time.date }}-{{ ansible_date_time.time }}
```

Comment:
In `rpcd/playbooks/roles/rpc_pre_upgrade/templates/status.txt.j2` that it correct, as that is just outputting to a text file and not running the command. 

It was incorrect in `rpcd/playbooks/roles/rpc_pre_upgrade/tasks/gather_stats.yml` which is probably what you meant to note.